src/lean_spec/subspecs/containers/signature.py

```diff
@@ -0,0 +1,62 @@
+"""Signature container."""
+
+from __future__ import annotations
+
+from lean_spec.types.collections import SSZList, SSZVector
+from lean_spec.types.container import Container
+from lean_spec.types.uint import Uint8
+
+MAX_TREE_DEPTH = 32
+
+
+class HashDigest(SSZVector):
+    """A 32-byte hash digest."""
+
+    ELEMENT_TYPE = Uint8
+    LENGTH = 32
+
+
+class Parameter(SSZVector):
+    """A 32-byte parameter."""
+
+    ELEMENT_TYPE = Uint8
+    LENGTH = 32
+
+
+class Randomness(SSZVector):
+    """A 32-byte randomness seed."""
+
+    ELEMENT_TYPE = Uint8
+    LENGTH = 32
+
+
+class HashDigestList(SSZList):
+    """A list of hash digests."""
+
+    ELEMENT_TYPE = HashDigest
+    LIMIT = MAX_TREE_DEPTH
```

Comment:
Assuming `HashDigestList` is for `signature.hashes` and not `signature.path`, this should be:

```suggestion
    LIMIT = XmssConfig.DIMENSION
```